config/karma.saucelabs.js

```diff
@@ -119,19 +114,50 @@ module.exports = function(config) {
 
     customLaunchers: sauceLabsBrowsers,
 
-    reporters: ['spec', 'saucelabs'],
+    reporters: ['spec', 'summary', 'saucelabs'],
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
 
     // concurrency: 10,
 
+    client: {
+      mocha: {
+        timeout: 20000,
+        retries: 3,
+        grep:
+          'Crawler Support|local timestamp|raises error event|handles failures',
```

Comment:
As somebody viewing this code without context, I'll be curious about why these tests are disabled and wonder if/when they should be re-enabled.  Can you please log bugs and link them here?  E.g.:

// TODO(b/123, b/456, ...): The tests matched by this regex are currently failing in the saucelabs cross-browser tests. They're disabled until we have a chance to investigate / fix them.

I'd also try to make this regex as specific as possible so that we don't accidentally match unrelated tests (or if we could somehow exclude the tests directly at the test definitions, that would be better than having a golbal regex).  Is it possible to include the full test name (including the describe() part)?  e.g. "EventManager raises error event" instead of just "raises error event" ?